package.json

```diff
@@ -41,8 +44,11 @@
     "eslint-plugin-import": "2.22.1",
     "istanbul": "0.4.5",
     "jasmine": "3.5.0",
-    "parse": "3.3.1",
+    "parse": "5.3.0",
```

Comment:
Please revert all upgrades that are not required for this PR

Reply:
Sure